### ISSUE TITLE
jshint warning on dev stack

### ISSUE THREAD

**Issue by nchapman** [mozilla/chronicle#273]
I don't see this locally, but it happens on the dev server:

```
Running "jshint:app" (jshint) task

server/db/postgres.js
  line 45  col 30  'query' is already defined.

  ⚠  1 warning

Warning: Task "jshint:app" failed. Used --force, continuing.
```

**Comment by pdehaan** [comment]
I do reckon this will be fixed in #283

**Comment by 6a68** [comment]
yup

**Comment by 6a68** [comment]
should be fixed by #283
